### DISCUSSION TITLE
Add arm64 architecture

### DISCUSSION THREAD

**Discussion by LucaTNT** [tiangolo/meinheld-gunicorn-docker#135] [Questions] [answered]
Hi,
I've been experimenting with Docker on ARM lately (both on my M1 Mac and on Graviton-based instances on AWS), and I had to manually build this (great!) image in order to get it working there.

It does not seem to have any issues, maybe you could add an "official" ARM version?

Thanks,
Luca

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile: 

There's been support for Arm for a while, but I didn't come here to comment then. :nerd_face:
